code/datums/supplypacks/munitions_vr.dm

```diff
@@ -3,8 +3,19 @@
 	contains = list(
 			/obj/item/weapon/gun/energy/locked/frontier = 2,
 			/obj/item/weapon/gun/energy/locked/frontier/holdout = 2,
+			/obj/item/weapon/gun/energy/locked/frontier/handbow=2
```

Comment:
If this is going to have its own crate, it should probably not be included in this crate.

Reply:
aparantly discard changes doesn't mean the same to be as it does to github